cinema/urls.py

```diff
@@ -1 +1,31 @@
-# write urls here
+from django.urls import path, include
+from rest_framework import routers
+
+from cinema.views import (
+    CinemaHallViewSet,
+    GenreViewSet,
+    ActorViewSet,
+    MovieViewSet,
+    MovieSessionViewSet,
+    OrderViewSet,
+    TicketViewSet
+)
+
+
+router = routers.SimpleRouter()
+
+
+router.register("cinema_halls", CinemaHallViewSet)
+router.register("genres", GenreViewSet)
+router.register("actors", ActorViewSet)
+router.register("movies", MovieViewSet)
+router.register("movie_sessions", MovieSessionViewSet)
+router.register("orders", OrderViewSet)
+router.register("tickets", TicketViewSet)
+
+
+app_name = "cinema"
+
+urlpatterns = [
+    path("", include(router.urls))
+]
```

Comment:
When basepath is empty no need to include, instead use:
urlpatterns = router.urls